draft-ietf-grow-rpki-as-cones.xml

```diff
@@ -146,28 +169,6 @@
 						</t>
 					</section>
 
-					<section title="ASN.1 format of a Policy Definition object">
-						<figure title="ASN.1 format of a Policy definition object">
-							<artwork><![CDATA[
-ASNPolicy DEFINITIONS ::=
-BEGIN
-Neighbours ::= SEQUENCE OF Neighbour
-
-Neighbour ::= SEQUENCE 
-{
-ASN INTEGER (1..42949672965),
-ASCone  VisibleString
-}
-
-Version ::= INTEGER
-LastModified ::= GeneralizedTime
```

Comment:
`LastModified` myabe is redundant, because the EE certificate's validity start date also exists?